src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.supliers.FigureSupplier;
+import java.util.Random;
+
+public class Main {
+    private static final int LENGTH = 5;
```

Comment:
First, make the title more informative. And secondly, according to the terms of the assignment, there must be 6 figures.